src/nvim/api/command.c

```diff
@@ -31,14 +31,15 @@
 /// @param[out] err  Error details, if any.
 /// @return Dictionary containing command information, with these keys:
 ///         - cmd: (string) Command name.
-///         - range: (array) Command <range>. Can have 0-2 elements depending on how many items the
-///                          range contains. Has no elements if command doesn't accept a range or if
-///                          no range was specified, one element if only a single range item was
-///                          specified and two elements if both range items were specified.
-///         - count: (number) Any |<count>| that was supplied to the command. -1 if command cannot
-///                           take a count.
-///         - reg: (string) The optional command |<register>|, if specified. Empty string if not
-///                         specified or if command cannot take a register.
+///         - range: (optional) (array) Command |<range>|. Omitted if command doesn't accept a range
+///                                     or if no range was specified.  Otherwise, has no elements if
+///                                     no range was specified, one element if only a single range
+///                                     item was specified, or two elements if both range items were
+///                                     specified.
```

Comment:
```suggestion
///         - range: (optional) (array) Command |<range>|. Omitted if command doesn't accept a range.
///                                     Otherwise, has no elements if no range was specified, one element if only
///                                     a single range item was specified, or two elements if both range items were
///                                     specified.
```